client/asset/dcr/dcr.go

```diff
@@ -1192,21 +1210,17 @@ func (dcr *ExchangeWallet) shutdown() {
 type compositeUTXO struct {
 	rpc   walletjson.ListUnspentResult
 	input *dexdcr.SpendInfo
+	confs int64
+	// TODO: consider including isDexChange bool for consumer
```

Comment:
`confs` is not used by the caller ([`fund`](https://github.com/decred/dcrdex/pull/499/files#diff-f9a1d582fb0d535d9f0525ab613f961dR425)), but I'm thinking one of the simple mitigations for DCR would be to pick the confirmed ones first, as stated in the TODO in that function.